docs/resources/aws_network_acls.md

```diff
@@ -0,0 +1,119 @@
+---
+title: About the aws_network_acls Resource
+platform: aws
+---
+
+# aws\_network\_acls
+
+Use the `aws_network_acls` InSpec audit resource to test the properties of all Amazon network ACLs.
+To audit a single network ACL, use the `aws_network_acl` (singular) resource.
+
+## Syntax
+
+An `aws_network_acls` resource block collects a group of network ACL descriptions and then tests that group.
+
+    describe aws_network_acls
+      it { should exist }
+    end
+
+### Parameters
+
+This resource does not expect any parameters.
+
+## Properties
+
+|Property                    | Description| Field Name |
+| ---                        | ---        | ---        |
+|network_acl_ids             | The IDs of all network ACLs.                 | network_acl_id |
+|vpc_ids                     | The IDs of all the VPCs of the network ACLs. | vpc_id |
+|default                     | The boolean flag to say whether the network ACL is default or not. | default |
+|default_network_acl_ids     | The default network ACL IDs for all the VPCs. | default_network_acl_ids |
+|associated_subnet_ids       | The ID of the associated subnets to all the network ACLs. | associated_subnet_ids |
```

Comment:
Field name should be associated_subnet_id. I think 's' should be not there. Same applies for most of the field names.

Reply:
I have [configured](https://github.com/inspec/inspec-aws/pull/339/files?file-filters%5B%5D=.rb&file-filters%5B%5D=.tf#diff-419b3280ad17035762c77f9ec151396bc795f04effb04904ad5f136e538315e2R73) the field name as `associated_subnet_ids` so that is what is described here and same goes for all other field names as well.. 

now those attributes are a collection for every single entity hence it is described in pluralised form